stylix/autoload.nix

```diff
@@ -1,17 +1,21 @@
 { lib, inputs }:
+rec {
+  # string -> { name = path; }
+  # List include path for either nixos modules or hm modules
+  attrs =
+    for:
+    lib.concatMapAttrs (
+      path: kind:
+      if kind == "directory" then
+        let
+          file = "${inputs.self}/modules/${path}/${for}.nix";
+        in
+        lib.optionalAttrs (builtins.pathExists file) { ${path} = file; }
```

Comment:
Since the docs say that adding a maintainer is required for new modules, should we remove this `pathExists` check and raise an error if `meta.nix` is missing? We'd need to keep the function as-is for autoloading other things, though.